app/assets/javascripts/users.js

```diff
@@ -65,6 +65,11 @@ WebsiteOne.define('Users', function () {
         });
 
         $('#require_users_profile').modal({show: true});
+
+        // Enable bootstrap tooltips on non-touch devices only
+        if (!('ontouchstart' in document.documentElement)) {
+            $('[data-toggle="tooltip"]').tooltip({delay: { "show": 500, "hide": 100 }});
```

Comment:
@tansaku No, it enables the tooltip and passes in an option to adjust the delay as part of the enabling: http://getbootstrap.com/javascript/#tooltips-options
You will see the tooltip working if you checkout  http://freeranger-wso-longnames.herokuapp.com/users